test/k8s-integration/cluster.go

```diff
@@ -269,31 +276,103 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string) err
 		}
 	}
 
-	var cmd *exec.Cmd
-	cmdParams := []string{"container", "clusters", "create", *gkeTestClusterName,
-		locationArg, locationVal, "--num-nodes", strconv.Itoa(numNodes),
-		"--quiet", "--machine-type", "n1-standard-2", "--image-type", imageType}
-	if isVariableSet(gkeClusterVer) {
-		cmdParams = append(cmdParams, "--cluster-version", *gkeClusterVer)
+	if useManagedDriver {
```

Comment:
nit: should really call this "useStagingDriver"; it will be useful maybe even after the managed driver is out.

Reply:
done.